renderdoc/driver/ihv/amd/amd_counters.cpp

```diff
@@ -26,80 +26,24 @@
 #include <Windows.h>
 #include "common/common.h"
 #include "core/plugins.h"
-#include "official/GPUPerfAPI/Include/GPUPerfAPI.h"
 #include "official/GPUPerfAPI/Include/GPUPerfAPIFunctionTypes.h"
+#include "official/GPUPerfAPI/Include/GPUPerfapi.h"
```

Comment:
Can you keep the case of the `#include` matching the case of the filename on disk? We might want to re-use this file in future for vulkan counters on linux e.g. and the GPA headers seem like they are intended to handle non-windows.